ethdb/pebble/pebble.go

```diff
@@ -347,7 +329,16 @@ func (d *Database) Has(key []byte) (bool, error) {
 	if d.closed {
 		return false, pebble.ErrClosed
 	}
+	st := time.Now()
 	_, closer, err := d.db.Get(key)
```

Comment:
I am not sure if it's worthwhile to measure every database read/write, honestly. Need to discuss with team